beets/plugins.py

```diff
@@ -94,6 +95,17 @@ def commands(self):
         """
         return ()
 
+    def get_early_import_stages(self):
+        """Return a list of functions that should be called as importer
+        pipelines stages early in the pipeline.
+
+        The callables are wrapped versions of the functions in
+        `self.early_import_stages`. Wrapping provides some bookkeeping for the
+        plugin: specifically, the logging level is adjusted to WARNING.
+        """
+        return [self._set_log_level_and_params(logging.WARNING, import_stage)
+                for import_stage in self.early_import_stages]
```

Comment:
Perhaps the logging wrapper code (here and in the function below) can be factored out into a helper function?

Reply:
Sure.